test/bases/renku_data_services/data_api/utils.py

```diff
@@ -7,6 +7,31 @@
 from renku_data_services.base_models import APIUser
 
 
+class AttributeDictionary(dict):
+    """Enables accessing dictionary keys as attributes"""
```

Comment:
I put it there because it was also in an `utils` file in `renku-notebooks` thinking it would be used as well for other tests but from the looks of it it won't be the case so I'll move it to `test_notebooks`